pom.xml

```
@@ -268,6 +271,67 @@
         </dependencies>
       </plugin>
 
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <version>${maven-shade-plugin.version}</version>
+        <configuration></configuration>
+        <executions>
+          <execution>
+            <id>shade-jar-with-dependencies</id>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <phase>package</phase>
+            <configuration>
+              <createDependencyReducedPom>false</createDependencyReducedPom>
+              <useDependencyReducedPomInJar>false</useDependencyReducedPomInJar>
+              <createSourcesJar>true</createSourcesJar>
+              <shadedArtifactAttached>true</shadedArtifactAttached>
+              <shadedClassifierName>shaded</shadedClassifierName>
+              <relocations>
+                <relocation>
+                  <pattern>org.apache</pattern>
+                  <shadedPattern>${shade.prefix}.org.apache.shaded</shadedPattern>
```

Should this be named like this for consistency with the other relocations?
```suggestion
                  <shadedPattern>${shade.prefix}.org.apache</shadedPattern>
```